fishtest/fishtest/static/html/SPRTcalculator.html

```diff
@@ -0,0 +1,102 @@
+<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Chess SPRT Calculator</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
+        <style>
+            td {
+                text-align: right;
+            }
+            .axis text {
+              font: 10px sans-serif;
+            }
+
+            .axis path, .axis line {
+              fill: none;
+              stroke: #000;
+              shape-rendering: crispEdges;
+            }
+
+            .line {
+              fill: none;
+              stroke: #000;
+              stroke-width: 2px;
+            }
+
+            .elo {
+              fill: none;
+              stroke: #000;
+              shape-rendering: crispEdges;
+            }
+
+            .grid-background {
+              fill: #ddd;
+            }
+
+            .grid line {
+              stroke: #fff;
+              stroke-width: 1px;
+              shape-rendering: crispEdges;
+            }
+
+            .grid .minor {
+              stroke-opacity: 0.25;
+            }
+
+            .grid text {
+              display: none;
+            }
+
+            .grid path,
+            .axis path {
+              display: none;
+            }
+
+            .form-control.number {
+              width: 4em;
+              text-align: right;
+              margin-right: 0.5em;
+            }
+
+            .chart-tooltip circle {
+                fill: white;
+                stroke: black;
+                stroke-width: 2px;
+            }
+
+        </style>
+    </head>
+    <body>
+        <div class=container>
+            <h1 class=page-header>Chess SPRT Calculator</h1>
+            <form id=parameters class=form-inline>
+                <div class=form-group>
+                    <label for=elo-0>Elo 0</label>
+                    <input id=elo-0 name=bayes-elo-0 value=-1 class='form-control number'>
```

Comment:
@vdbergh is the name (bayes-elo-0) still correct?

Reply:
Thanks. The script uses getElementById it seems so the name of the field is not actually important. But of course it is not nice. I will change it (later during the day).

Reply:
PR updated.

Reply:
git push ?

Reply:
Pushed to the wrong branch. Fixed now.